airbyte-cdk/python/unit_tests/destinations/test_destination.py

```diff
@@ -180,6 +180,7 @@ def test_run_check(self, mocker, destination: Destination, tmp_path):
         parsed_args = argparse.Namespace(**args)
         destination.run_cmd(parsed_args)
 
+        mocker.patch.object(destination, "spec", return_value=ConnectorSpecification(connectionSpecification={}))
```

Comment:
we should add a test case here too to verify the config is validated. 

It may help to look at test coverage by running `./type_check_and_test.sh`

Reply:
There is already detailed test case that covers validation function here: https://github.com/airbytehq/airbyte/pull/5457/files#diff-0d9087e19f5bdabe137529bb861aaaa261b39d5344e79041602a2a94c7530301R150 So Ive just added checks if this validation function is called with correct args for destination.

Reply:
Thanks, done